### ISSUE TITLE
Reproducibility Assisstance

### ISSUE THREAD

**Issue by kanlions** [mahmoodlab/CONCH#24]
Thank you for great research work and making it available for research community. I am particularly working on the zero shot classification. I have worked with the demo code  in zero shot starter. The datasets are CRC100k (colorectal cancer tissue classification), WSSS4LUAD (LUAD tissue classification) and SICAP (Gleason pattern classification). If I am correct I see the numbers in .75 ish range and I am getting way off. Can you please confirm is it the same prompts which is available in prompts folder. Also it would be very kind if you can also tell me whether I am using correct set CRC-VAL-HE-7K, (with 8 classes), WSS4luad (training 1.14gb, val 150mb from Grand challenges)  and SICAPv2 (18793 images). If any discrepancy please guide me to the correct prompts files for these 3 datasets and also data sources. It is kind of important for my understanding and future use of this model that the correct baseline is established and these 3 datasets work uniformly for the associated prompt.

Thanks in advance

**Comment by kanlions** [comment]
Thank you very much for your reply. If I understand there are two results one is in main paper Figure 2 (c) which I am talking about. And in extended supplementary Figure 2 extended where for the three datasets are shown. Since I am struggling with reproducibility, can you please provide the following information about what subsets of images for the three datasets was used for the results and also prompts as for SICAP only primary and secondary gleason score is available. It will be very helpful if the relevant prompts of the paper are uploaded in Github as it is a VLM model so text cues may have a significant impact. My first goal is to run reproduce your method to establish baseline. As in future when I run on new set of images and cite your paper I want to be sure that correct results are generated and communicated. Again thanks for your help.

**Comment by fedshyvana** [comment]
Prompts are listed in supplementary tables 38 - 44. The splits are described in Methods section under Downstream evaluation datasets (CRC-100k = validation set, WSS4LUAD = subset of training set, SICAP = test set).

**Comment by kanlions** [comment]
Thanks a ton for your input and apologies for my oversight in the supplementary material. With respect to WSSS4LUAD the without ensembling values are higher values. Supplementary tables 7 and 14. Isnt this behavior abnormal? It seems all competing methods show it? Anyways thanks, I will run experiments to see

**Comment by fedshyvana** [comment]
7 is ensembling and 14 is no ensembling, values are consistently much higher in 7, for all methods right?

**Comment by kanlions** [comment]
Thanks a lot for your support and apologies for the confusion. So currently I generated the json files based on the supplementary material tables and running the experiments. Though still not matching but I am investigating from my end. Anyways thanks for all the clarifications
